tests/conftest.py

```diff
@@ -2,14 +2,15 @@
 import shutil
 
 import pytest
-from flask import current_app
+from flask import Flask, current_app
 
 from app import create_app
 
 
 @pytest.fixture(scope='session')
 def notify_ftp():
-    app = create_app()
+    app = Flask('app')
```

Comment:
Same as above ftp-app?

Reply:
changed this to `test` cos its a test app